mgmt/rest/v2/plugin.go

```diff
@@ -40,49 +41,114 @@ import (
 	"github.com/julienschmidt/httprouter"
 )
 
+// PluginResponse represents the response from plugin operations.
+//
+// swagger:response PluginResponse
+type PluginResponse struct {
+	// List of a plugin
+	//
+	// in: body
+	Plugin Plugin `json:"plugin,omitempty"`
+}
+
+// PluginsResp represents the response from plugins operations.
+//
+// swagger:response PluginsResponse
+type PluginsResp struct {
+	// List of plugins
+	//
+	// in: body
+	Body struct {
+		Plugins []Plugin `json:"plugins,omitempty"`
+	}
+}
+
 type PluginsResponse struct {
-	RunningPlugins []RunningPlugin `json:"running_plugins,omitempty"`
-	Plugins        []Plugin        `json:"plugins,omitempty"`
+	Plugins []Plugin `json:"plugins,omitempty"`
 }
 
+// Plugin represents a plugin type definition.
 type Plugin struct {
-	Name            string        `json:"name"`
-	Version         int           `json:"version"`
-	Type            string        `json:"type"`
-	Signed          bool          `json:"signed"`
-	Status          string        `json:"status"`
-	LoadedTimestamp int64         `json:"loaded_timestamp"`
-	Href            string        `json:"href"`
-	ConfigPolicy    []PolicyTable `json:"policy,omitempty"`
+	Name             string        `json:"name"`
+	Version          int           `json:"version"`
+	Type             string        `json:"type"`
+	Signed           bool          `json:"signed"`
+	Status           string        `json:"status"`
+	LoadedTimestamp  int64         `json:"loaded_timestamp,omitempty"`
+	Href             string        `json:"href,omitempty"`
+	ConfigPolicy     []PolicyTable `json:"config_policy,omitempty"`
+	HitCount         int           `json:"hitcount,omitempty"`
+	LastHitTimestamp int64         `json:"last_hit_timestamp,omitempty"`
+	ID               uint32        `json:"id,omitempty"`
+	PprofPort        string        `json:"pprof_port,omitempty"`
+}
```

Comment:
Merged `RunningPlugin` with 'Plugin` data type. The difference can be omitted in the JSON representation in terms of different plugin response scenarios. The reason is to simplify the plugin data model in swagger spec.